src/reason_pprint_ast.ml

```diff
@@ -697,6 +699,16 @@ let rules = [
   [
     (TokenPrecedence, (fun s -> (Nonassoc, s = "=>")));
   ];
+  [
+    (TokenPrecedence, (fun s ->
+      let is_infix_op = Hashtbl.mem infix_operators s in
```

Comment:
I thought about similar to Haskell's `infix`, `infixl` and `infixr` mechanism. To allow user to choose associativity of operator.